packages/extension-chakra-storefront/src/components/forms/credit-card-fields.jsx

```diff
@@ -131,33 +113,31 @@ const CreditCardFields = ({form, prefix = ''}) => {
                     <Field
                         {...fields.securityCode}
                         formLabel={
-                            <>
-                                <FormLabel display="inline" mr={1}>
-                                    {fields.securityCode.label}
-                                </FormLabel>
-                                <Box
-                                    onMouseEnter={handleTooltipOpen}
-                                    onFocus={handleTooltipOpen}
-                                    as="span"
-                                >
+                            <ChakraField.Label>
+                                <Flex align="center" justify="space-between">
+                                    <Box>{fields.securityCode.label}</Box>
                                     <Tooltip
```

Comment:
For ease of use, created a closed component composition for the Tooltip component as suggest in the Chakra doc